nodejs/week3/mealshare/src/backend/api/meals.js

```diff
@@ -0,0 +1,170 @@
+const { request, response } = require("express");
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+// implementing GET
+
+router.get("/", async (request, response) => {
```

Comment:
Good job! 👍 
As an extra, you could think about what should happen if there are no meals with that title, or no meals with the given max Price. (So if you get an empty array as a response.)